src/nxdoc/nuxeo-add-ons/nuxeo-enhanced-viewer.md

```diff
@@ -338,78 +330,189 @@ A new window is opened with the two documents side-by-side, highlighting text wh
+In the end, you should have the following in your `nuxeo.conf`:
 
-- For a Docker installation, you can extend our image and copy your properties file to `/docker-entrypoint-init.d/arender.properties`:
+```properties
+arender.server.previewer.host=https://arender-previewer-url
+nuxeo.arender.oauth2.client.create=true
+nuxeo.arender.oauth2.client.id=arender
+nuxeo.arender.oauth2.client.secret=OAUTH2_SECRET
+nuxeo.arender.oauth2.client.redirectURI=/login/oauth2/code/nuxeo
+```
 
-  ```
-  FROM dockerin-arender.nuxeo.com:443/arender-previewer:MP_VERSION
+### ARender Configuration
 
-  COPY arender.properties /docker-entrypoint-init.d/arender.properties
-  ```
+Nuxeo Enhanced Viewer involves to install the ARender services. You can install the ARender services using Kubernetes and Helm 3 by following [ARender Documentation](https://arender.io/v4/documentation/install/kubernetes/).
 
-You can also modify the `arender-hmi.properties` which is deployed inside the `/ARenderConfiguration` in the previewer Docker container.
+{{#> callout type='warning' heading='Private addon'}}
+You should contact your Nuxeo Administrator or your Nuxeo sales representative to get access to these artefacts.
+{{/callout}}
+
+#### Common
+
+After getting the ARender Helm Chart, you need to customize some part in order to deploy the Nuxeo connector and add configuration. You can find below an example of _values.yaml_:
+
+```yaml
+web-ui:
+  image:
+    repository: docker-arender.packages.nuxeo.com/nuxeo/arender-previewer
+    tag: [NEV_VERSION]
+  environment:
+    # required values
+    ARENDERSRV_NUXEO_SERVER_URL: https://nuxeo-host/nuxeo
+    ARENDERSRV_NUXEO_SERVER_ARENDER_SECRET: OAUTH2_SECRET # Same than the one in nuxeo.conf
+    # optional values with their defaults
+    ARENDERSRV_NUXEO_CLIENT_TIMEOUT: 30
+    ARENDERSRV_NUXEO_SERVER_CONTEXT_PATH: ''
+rendition:
+  broker:
+    image:
+      repository: docker-arender.packages.nuxeo.com/arender-document-service-broker
+      tag: [ARENDER_VERSION]
+  converter:
+    image:
+      repository: docker-arender.packages.nuxeo.com/arender-document-converter
+      tag: [ARENDER_VERSION]
+    environment:
+      TKC_TOOLS_IMAGEMAGICK_OPTIONS: "-quality,100,-density,72x72,-units,PixelsPerInch,-auto-orient,+repage"
+      TKC_IMAGE_CONVERSION_TIMEOUT: "120"
+  dfs:
+    image:
+      repository: docker-arender.packages.nuxeo.com/arender-document-file-storage
+      tag: [ARENDER_VERSION]
+  renderer:
+    image:
+      repository: docker-arender.packages.nuxeo.com/arender-document-renderer
+      tag: [ARENDER_VERSION]
+  handler:
+    image:
+      repository: docker-arender.packages.nuxeo.com/arender-document-text-handler
+      tag: [ARENDER_VERSION]
+```
+
+{{#> callout type='warning' heading='Renaming in 10.3'}}
+
+Before 10.3:
+
+- `ARENDERSRV_NUXEO_SERVER_URL` was named `ARENDERSRV_NUXEO_URL`
+- `ARENDERSRV_NUXEO_SERVER_ARENDER_SECRET` was named `ARENDERSRV_NUXEO_ARENDER_SECRET`
```

Comment:
Worth keeping backward compatibility?

Reply:
There's backward compatibility in 10.3.x.

As the 10.2 will be end of life soon, I wanted to make the configuration for 10.3 the first citizen.